### ISSUE TITLE
How to achieve the desired verification accuracy？

### ISSUE THREAD

**Issue by emc2-2022** [titu1994/Wide-Residual-Networks#6]
I was using the default settings (with Tensorflow backend) but my test accuracies are worse:

For WRN-16-8 put N = 2, k = 8
@ Epoch 82/100: - 184s - loss: 0.0711 - acc: 0.9768 - val_loss: 0.4480 - val_acc: 0.9187
For WRN-28-10 put N = 4, k = 10
@ Epoch119/200 - 378s - loss: 0.0260 - acc: 0.9915 - val_loss: 0.4396 - val_acc: 0.9213

There seems to be a severe overfitting (each model contains more than 10 million trainable parameters).
I was wondering what can I do to achieve the desired accuracy? 
Do I need to change the preprocessing settings? or adjust the learning rate? or reduce filters for each layer? ...
Please~

**Comment by titu1994** [comment]
Are you training the new model types, or the old ones? The old ones have lower performance by 1 - 1.1% compared to the new ones.

Other than that, I haven't trained the newer models yet but for the older models I would save the weights after 1 training, use a smaller learning rate and train again for 50 iterations or so and then continue this until no more improvements were seen.

**Comment by emc2-2022** [comment]
Thanks for your reply！
I was using the current code you updated recently. So I think they are the 'new' model types. I have tried many times but can't achieve the accuracy of 93.68% or 95.19%(for WRN-16-8 put N = 2, k = 8).
Then I tried to use the callbacks.LearningRateScheduler (details: switch to SGD optimizer, start with a learning rate of 0.002, divide it by 5 at the 50th epoch, the 75th epoch , and the 100th epoch and terminate training at the 140th epoch). The accuracy was locked between 92% and 92.7% after the 100th epoch.

Can you provide the optimizer settings? (SGD or Adam? the value of the smaller learning rate?) Please~

(Besides, with my experience, the vgg16 style net can also reach to the accuracy of 92%. Wide Residual Networks should be better than that. So I will keep on trying.)

**Comment by titu1994** [comment]
Sorry for the late reply. I used multiple restarts to achieve that score on the older model. With multiple restarts, basically, rerun the code (load weights from previous best run, use somewhat high initial learning rate (1e-4, 1e-5) on Adam optimizer. 

It was hard to get that score, as I remember it, and required many many restarts to get to diminishing returns. I think I stopped after 10-12 runs like this.

I haven't tried the new models. I got a reply in the issues section, stating that the newer model config provided a 1% boost in the score. Though I didn't verify this claim, the model now resembles the Torch implementation precisely (by the authors I believe), so I updated the models accordingly. 

If this turns our to be wrong, I will revert to the previous version.
